crates/ruff/src/rules/flake8_boolean_trap/rules/boolean_default_value_in_function_definition.rs

```diff
@@ -11,18 +11,18 @@ use crate::rules::flake8_boolean_trap::helpers::{add_if_boolean, is_allowed_func
 /// Checks for the use of booleans as default values in function definitions.
 ///
 /// ## Why is this bad?
-/// Calling a function with boolean default means that the keyword argument
-/// argument can be omitted, which makes the function call ambiguous.
+/// Calling a function with boolean default means that the argument argument
+/// can be omitted, which makes the function call ambiguous.
 ///
-/// Instead, consider defining the relevant argument as a required keyword
+/// Instead, consider defining the relevant argument as a _required_ keyword
 /// argument to force callers to be explicit about their intent.
 ///
 /// ## Example
 /// ```python
 /// from math import ceil, floor
 ///
 ///
-/// def round_number(number: float, up: bool = True) -> int:
+/// def round_number(number: float, *, up: bool = True) -> int:
```

Comment:
Does this example trigger the rule?

Reply:
No, that was part of my confusion. The two rules now use similar documentation, and make it clear that the problem is just around positional arguments broadly, not default values specifically.